webauthn.js

```diff
@@ -237,6 +237,7 @@ navigator.credentials = navigator.credentials || (function () {
 			}
 
 			return getCredList(allowlist).then((credList) => {
+				const challenge = publicKeyCredRequest.challenge;

```

Comment:
I haven't updated the challenge portion yet because the msCredential expects a DOMString but the new spec makes it a ArrayBuffer. Some conversion needs to be done. This work is on my plate for this week.